tests/testthat/test-pmap.R

```diff
@@ -66,21 +89,7 @@ test_that("preserves S3 class of input vectors (#358)", {
   expect_output(pwalk(list(date), print), format(date))
 })
 
-test_that("walk2() and pwalk() don't evaluate symbolic objects", {
-  walk2(exprs(1 + 2), NA, ~ expect_identical(.x, quote(1 + 2)))
-  pwalk(list(exprs(1 + 2)), ~ expect_identical(.x, quote(1 + 2)))
-})
-
-test_that("map2() and pmap() don't evaluate symbolic objects", {
-  map2(exprs(1 + 2), NA, ~ expect_identical(.x, quote(1 + 2)))
+test_that("don't evaluate symbolic objects (#428)", {
   pmap(list(exprs(1 + 2)), ~ expect_identical(.x, quote(1 + 2)))
-})
-
-test_that("pmap() with empty input copies names", {
-  named_list <- list(named(list()))
-  expect_identical(    pmap(named_list, identity), named(list()))
-  expect_identical(pmap_lgl(named_list, identity), named(lgl()))
-  expect_identical(pmap_int(named_list, identity), named(int()))
-  expect_identical(pmap_dbl(named_list, identity), named(dbl()))
-  expect_identical(pmap_chr(named_list, identity), named(chr()))
+  pwalk(list(exprs(1 + 2)), ~ expect_identical(.x, quote(1 + 2)))
 })
```

Comment:
Maybe a `pmap_vec()` test about enforcing `.ptype` (which would catch what Lionel pointed out before about missing the ptype arg)